test/runtests.jl

```diff
@@ -783,3 +783,7 @@ end
 @testset "Plot recipes" begin
     include("plots.jl")
 end
+
+@testset "JuliaLang/julia#30944" begin
```

Comment:
Maybe add the tests there were in #55 and call this testset "Ranges"?

Reply:
Just seemed a bit redundant, since they all just end up calling `range(a, step=b, length=l)`, but probably doesn't hurt much either

Reply:
It's an integration test :slightly_smiling_face: